locations/categories.py

```diff
@@ -796,6 +803,62 @@ def apply_clothes(clothes: [Clothes], item: Feature):
         apply_category({"clothes": c.value}, item)
 
 
+class Vending(Enum):
+    """
+    https://wiki.openstreetmap.org/wiki/Key:vending
+    """
+
+    BICYCLE_TUBE = "bicycle_tube"
+    BOTTLE_RETURN = "bottle_return"
+    COFFEE = "coffee"
+    DRINKS = "drinks"
+    FOOD = "food"
+    KEYS = "key"
+    LAUNDRY = "laundry"
+    PARKING_TICKETS = "parking_tickets"
+
+
+def add_vending(vending: Vending | list[Vending], item: Feature):
+    if item["extras"].get("vending"):
+        current = item["extras"]["vending"].split(";")
+    else:
+        current = []
+
+    for v in vending if isinstance(vending, list) else [vending]:
+        if v.value not in current:
+            current.append(v.value)
+
+    item["extras"]["vending"] = ";".join(current)
```

Comment:
It would be nice to factor out the "check extras and apply enum" code so that it can be used for other types of categories as well.